API.md

```diff
@@ -39,6 +41,30 @@ See also [`reject`](#reject), [`resolve`](#resolve).
 all([ Promise.resolve('a') ]) //=> Promise ['a']
 ```
 
+### assocWith
+
+```haskell
+// assocWith : String -> ({ k: v } -> a) -> { k: v }
```

Comment:
I'm sure they were just copied from the the js comments, but the `//` don't belong in signatures.  Also, you're missing a final `-> { k: v }` to denote the return value type.